android-todo-list/architecture-samples-main/app/src/main/java/com/example/android/architecture/blueprints/todoapp/util/StatsIgUtil.kt

```diff
@@ -33,4 +34,13 @@ object StatsIgUtil {
         return Statsig.getConfig(TEST_MOBILE_DYNAMIC_CONFIG)
     }
 
+    /**
+     * Feature Gate function to check if access to delete of to-do list is granted.
+     *
+     * @return Boolean value of Feature Gate of FEATURE_GATE_DELETE_TODO_ACCESS.
+     */
+    fun isDeleteTodoFeatureEnabled(): Boolean {
+        return Statsig.checkGate(FEATURE_GATE_DELETE_TODO_ACCESS)
+    }
```

Comment:
this is a lot of unnecessary code - remove the constant, remove this method, and just add `Statsig.checkGate` inline in the method

Reply:
Okay. I have removed the StatsigUtil and replace with Statsig.checkGate

Reply:
Done in https://github.com/statsig-io/samples/pull/5